conf/supervisord-dev.conf

```diff
@@ -0,0 +1,35 @@
+[supervisord]
+nodaemon = true
+silent = true
+
+[program:web]
+command=gunicorn --paste conf/development.ini
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+
+[program:assets]
+command = node_modules/.bin/gulp watch
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+
+[eventlistener:logger]
+command=bin/logger --dev
+buffer_size=100
+events=PROCESS_LOG
+stderr_logfile=/dev/fd/1
+stderr_logfile_maxbytes=0
+stdout_logfile=/dev/null
+
+[unix_http_server]
+file = .supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl = unix://.supervisor.sock
+prompt = lms
```

Comment:
These three sections are all needed to get `supervisorctl` working. They tell `supervisord` to listen on `.supervisor.sock` using its RPC interface and `supervisorctl` to talk to `supervisord` on `supervisor.sock`